recipes/effolkronium-random/all/test_package/CMakeLists.txt

```
@@ -1,9 +1,7 @@
-cmake_minimum_required(VERSION 3.1)
-project(test_package)
+cmake_minimum_required(VERSION 3.15)
+project(test_package LANGUAGES CXX)
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+find_package(effolkronium_random REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
-target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
-set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
+target_link_libraries(${PROJECT_NAME} PRIVATE effolkronium_random)
```

I think the fail in the ci is because this is missing:
```suggestion
target_link_libraries(${PROJECT_NAME} PRIVATE effolkronium_random)
target_compile_features(${PROJECT_NAME} PUBLIC cxx_std_11)
```